docs/x509/reference.rst

```diff
@@ -1112,6 +1124,12 @@ X.509 CSR (Certificate Signing Request) Builder Object
         <NameAttribute(oid=<ObjectIdentifier(oid=2.5.4.10, name=organizationName)>, value=u'Test Certificates 2011')>
         <NameAttribute(oid=<ObjectIdentifier(oid=2.5.4.3, name=commonName)>, value=u'Good CA')>
 
+    .. attribute:: rdns
```

Comment:
Iterating `RelativeDistinguishedName` still yields `NameAttribute` values, for compatibility (and it still adequately covers the 99%** use case where each RDN has exactly one attribute).

So ``rdns`` lets you get the sequence of actual RDNs, if you need them.

** guessing :)